torchrl/envs/libs/smacv2.py

```diff
@@ -0,0 +1,620 @@
+# Copyright (c) Meta Platforms, Inc. and affiliates.
+#
+# This source code is licensed under the MIT license found in the
+# LICENSE file in the root directory of this source tree.
+import importlib
+from typing import Dict, Optional
+
+import torch
+from tensordict import TensorDict, TensorDictBase
+
+from torchrl.data import (
+    BoundedTensorSpec,
+    CompositeSpec,
+    DiscreteTensorSpec,
+    OneHotDiscreteTensorSpec,
+    UnboundedContinuousTensorSpec,
+)
+from torchrl.envs.common import _EnvWrapper
+
+
+_has_smacv2 = importlib.util.find_spec("smacv2") is not None
+if _has_smacv2:
+    import smacv2
+    from smacv2.env.starcraft2.maps import smac_maps
```

Comment:
do you mean that is should import it in every function that needs it?

Reply:
yeah the problem is that at some point I would like to expose all the wrappers in `torchrl.envs`.
If you have installed multiple third-party libs, you'll end up with a launch time that is really high. Moreover, our multirproc objects spawns processes that all need to import all of these things, which makes our tests really really long!
TLDR: no global imports for third party unless needed elsewhere or import is too frequent
(since local import has some overhead when repeated multiple times)